<ui version="4.0" >
 <class>NepomukSideBarItem</class>
 <widget class="QWidget" name="NepomukSideBarItem" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Rating:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KRatingWidget" native="1" name="mRatingWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Description:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>mDescriptionLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="KLineEdit" name="mDescriptionLineEdit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Tags:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="Gwenview::TagWidget" native="1" name="mTagWidget" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KRatingWidget</class>
   <extends>QWidget</extends>
   <header>nepomuk/kratingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Gwenview::TagWidget</class>
   <extends>QWidget</extends>
   <header>lib/metadata/tagwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
